package.json

```diff
@@ -391,6 +391,11 @@
 					"type": "boolean",
 					"default": false,
 					"description": "Whether to automatically send a 'hot reload' request during a Flutter debug session when saving files."
+				},
+				"dart.analyzeAngularTemplates": {
+					"type": "boolean",
+					"default": false,
```

Comment:
The release notes won't exist for this yet - you're free to create a placeholder in CHANGELOG but otherwise I can do it when preparing a release (I go through the Issues + Changesets in a release when compiling the notes).

I do think we should remove it from packages.json to hide it for now though, at least until the ability to use it is simpler (I presume "building it yourself" isn't the long-term plan!).